find-doi-requests/src/test/java/no/unit/nva/doi/requests/FindDoiRequestsHandlerTest.java

```diff
@@ -89,6 +89,33 @@ public void handleRequestReturnsStatusOKOnValidCreatorRoleInput() throws Excepti
         assertEquals(expected, actual);
     }
 
+    @Test
+    public void handleRequestReturnsStatusOKOnValidUpperCaseCreatorRoleInput() throws Exception {
+        when(doiRequestsService.findDoiRequestsByStatusAndOwner(
+            any(URI.class), any(DoiRequestStatus.class), anyString()
+        )).thenReturn(new DoiRequestsResponse());
+
+        InputStream inputStream = new HandlerRequestBuilder<Void>(objectMapper)
+            .withHeaders(getRequestHeaders())
+            .withQueryParameters(Map.of(ROLE, CREATOR.toUpperCase()))
+            .withRequestContext(getRequestContext())
+            .build();
+
+        handler.handleRequest(inputStream, outputStream, context);
+
+        GatewayResponse<DoiRequestsResponse> actual = objectMapper.readValue(
```

Comment:
refactor to `GatewayResponse<DoiRequestsResponse> actual = GatewayResponse.fromOutputStream(outputStream);`